app/src/test/java/lt/vilnius/tvarkau/base/BaseRobolectricTest.kt

```diff
@@ -0,0 +1,43 @@
+package lt.vilnius.tvarkau.base
+
+import android.support.v4.app.Fragment
+import lt.vilnius.tvarkau.TestActivity
+import org.junit.After
+import org.junit.Before
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.shadows.ShadowApplication
+import org.robolectric.util.ActivityController
+
+
+/**
+ * @author Martynas Jurkus
+ */
+@RunWith(RobolectricTestRunner::class)
+abstract class BaseRobolectricTest {
+
+    protected lateinit var activity: TestActivity
+    protected var activityController: ActivityController<TestActivity>? = null
+
+    @Before
+    open fun setUp() {
+        ShadowApplication.getInstance().declareActionUnbindable("com.google.android.gms.analytics.service.START")
```

Comment:
Ups. Copy paste from other project.
Basicly it disables services if there are any.